app/src/androidTest/java/com/mapbox/maps/testapp/annotation/UpdateAnnotationTest.kt

```diff
@@ -34,8 +31,8 @@ class UpdateAnnotationTest : BaseMapTest(), OnMapLoadedListener {
   private lateinit var runnable: Runnable
   private var index = 0
   private val latch = CountDownLatch(AnnotationUtils.STYLES.size * 3)
-  private lateinit var symbolManager: SymbolManager
-  private lateinit var symbol: Symbol
+  private lateinit var symbolManager: PointAnnotationManager
```

Comment:
shall we update symbolManager to be pointManager?

Reply:
Correct.